compiler/rustc_hir/src/hir.rs

```diff
@@ -2215,6 +2206,9 @@ impl PrimTy {
         Self::Str,
     ];
 
+    /// [`PrimTy::name`], but returns a &str instead of a symbol.
+    ///
+    /// Used by rustdoc.
```

Comment:
These "used by" comments will likely become outdated in the future.

Reply:
I think that's ok and they're still useful. If they become outdated because no one uses them, then it gives a hint that the function is dead code. If they become outdated because someone new uses them, then the comment is still accurate, just not complete.